src/main/java/org/fundacionjala/coding/nestor/Main.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+/**
+ * Main class to run this project.
+ */
+final class Main {
+  public final static int DAYS_RENTER = 3;
+  private final static Logger logger = Logger.getLogger("");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1312)